guides/doc-Provisioning_Guide/topics/Cloud-Azure.adoc

```diff
@@ -185,43 +187,48 @@ You can add Azure details to a compute profile to automatically populate virtual
 --compute-resource _azure_cr_name_ \         <2>
---organization "_Your_Organization_" \       <3>
---location "_Your_Location_" \               <4>
---compute-attributes="resource_group=_resource_group_,vm_size=_Standard_B1s_,username=_azure_user_,password=_azure_password_,platform=Linux" \ <5>
---interface="compute_public_ip=Dynamic,compute_network=mysubnetID,compute_private_ip=false" <6>
+--compute-attributes="resource_group=_resource_group_,vm_size=_Standard_B1s_,username=_azure_user_,password=_azure_password_,platform=Linux,script_command=touch /var/tmp/text.txt" \ <3>
+--interface="compute_public_ip=Dynamic,compute_network=mysubnetID,compute_private_ip=false" <4>
 ----
 <1> With the `--compute-profile` option, enter the name of a compute profile to configure.
 <2> With the `--compute-resource` option, enter a name of the Azure Compute resource that you want to configure a compute profile for.
-<3> With the `--organization` option, enter an organization to use.
-<4> With the `--location` option, enter a location to use.
-<5> With the `--compute-attributes` option, specify the following details:
+<3> With the `--compute-attributes` option, specify the following details:
 * With the `resource_group` setting, enter the resource group to provision to.
 * With the `vm_size` setting, enter the size of the VM to provision, for example `Standard_B1s`.
 * With the `username` setting, enter the SSH user name for image access. Note that the username that you enter for compute profile must be the same that you use when creating an image.
 * With the `password` setting, enter the password to authenticate with.
 * With the `platform` setting, enter `Linux`.
-<6> With the `--interface` option, specify the following details:
+* With the `script_command` setting, enter a command to be executed on the provisioned VM.
+<4> With the `--interface` option, specify the following details:
 * With the `compute_public_ip` setting, enter the public IP address type. Must be either `None`, `Static`, or `Dynamic`.
 * With the `compute_network` setting, enter the ID of an available Azure Subnet to use for provisioning.
 * With the `compute_private_ip` setting, specify whether to use a private IP address. Must be `true` or `false`.
 
 [[Provisioning_Virtual_Machines_in_Azure-Creating_Image_Based_Hosts_on_Azure]]
 === Creating Image-Based Hosts on Azure
 
-The Azure provisioning process creates hosts from existing images on Azure.
+The Azure provisioning process creates hosts from existing images on Azure. 
+
+.Static Private IP Considerations
+If you want the VM to use a static private IP, you must complete the following steps:
+
+. Ensure that you select the *Static Private IP* check box in the compute profile.
+. Create a subnet in Satellite with *Network Address* matching the Azure subnet.
+. While creating a host, on the *Network Interfaces* tab, select the created subnet and enter the IPv4 address within the range of your subnet.
```

Comment:
```suggestion
. While creating a host, on the *Network Interfaces* tab, select the created subnet in addition to the Azure subnet and enter the IPv4 address within the range of your subnet.
```

Reply:
Does that mean that on the *Interfaces* tab I need to add two interfaces?  For one inteface I can only choose one IPv4 Subnet.

Reply:
No, not two interfaces. Just select subnet at two places, like this:

![imageedit_2_7562814697](https://user-images.githubusercontent.com/9948256/74639948-c3310200-5194-11ea-9000-2ae4ab484afa.gif)